### ISSUE TITLE
CREMA-D: Why 90% of the Test set is used and 10% for Val?

### ISSUE THREAD

**Issue by kkontras** [lihong2303/AGM_ICCV2023#1]
Hi,

Thank you very much for sharing the code for the paper, it helps a lot. I noticed that in crema-d loader you have the lines:
`self.item = self.test_item[:int(len(self.test_item) * 0.9)]`
`self.image = self.test_image[:int(len(self.test_image) * 0.9)]`
`self.audio = self.test_audio[:int(len(self.test_audio) * 0.9)]`
`self.label = self.test_label[:int(len(self.test_label) * 0.9)]`
            
Is there any specific reason you are keeping the 90% of the test set, and you take that 10% for the validation set?

**Comment by lihongcs** [comment]
In our paper, we need to compute a linear predictor for a mono-modal concept. Therefore, we split 10% of the data from the train set and the test set to train the linear predictor.

**Comment by kkontras** [comment]
So if I understand well then the validation results that you print at the end of each epoch are on the `task.test_dataloader`, is that right?

Also if I may ask for a clarification, the linear predictor on top of the features is only to calculate the modality competition strength or is it actively used in the training procedure?

**Comment by ccw122** [comment]
> 所以如果我理解得很好，那么你在每个 epoch 结束时打印的验证结果都在 上，对吗？`task.test_dataloader`
> 
> 另外，如果我可以要求澄清一下，功能之上的线性预测器只是为了计算模态竞争强度，还是在训练过程中积极使用？

i have the same question,do you have the answer please?
